### DISCUSSION TITLE
What is next?

### DISCUSSION THREAD

**Discussion by timunie** [MahApps/MahApps.Metro#4000] [Ideas]
Hi together, 

Let's try the Discussions-section. If I find some free time, which new feature should I work on? 

- `MultiSelectionComboBox` #3802  ► Vote with 🚀 
- `ProgressRing` with support for `Content` and useable as a `PieChart` ► Vote with 🎉 
- Update some documentation ► Vote with 👀 and please write in the comments which control.

Thank you and happy coding
Tim

**Comment by punker76** [comment]
First and last now done. Second is a nice idea, I will thought about it.
